### PR TITLE
Fix options name for i3color-lock release 2.13.c.3

### PR DESCRIPTION
**Pull request by nicolas-s**
With release 2.13.c.3 i3color-lock has added dashes before color, pos, size, etc.
https://github.com/Raymo111/i3lock-color/releases/tag/2.13.c.3

### CODE REVIEW

**Review by medoix — APPROVED** [review]
I just came across this myself, thanks for raising this PR with the fix.

File: multilockscreen [hunk 1]
```diff
@@ -98,16 +98,16 @@ lock() {
...
+		--layout-pos='ix-90:iy-0' --layout-align 1 --layout-color=$layoutcolor $keylayout \
+		--clock --date-align 1 --date-str "$locktext" \
+		--inside-color=$insidecolor --ring-color=$ringcolor --line-uses-inside \
+		--keyhl-color=$keyhlcolor --bshl-color=$bshlcolor --separator-color=$separatorcolor \
+		--insidever-color=$insidevercolor --insidewrong-color=$insidewrongcolor \
+		--ringver-color=$ringvercolor --ringwrong-color=$ringwrongcolor --ind-pos='x+280:y+h-70' \
+		--radius=20 --ring-width=4 --verif-text='' --wrong-text='' --wrong-color="$nocolor" \
+		--verif-color="$verifcolor" --time-color="$timecolor" --date-color="$datecolor" \
 		--time-font="$font" --date-font="$font" --layout-font="$font" --verif-font="$font" --wrong-font="$font" \
 		--noinputtext='' --force-clock --pass-media-keys --pass-screen-keys --pass-power-keys \
```
Comment (medoix) [inline multilockscreen]: this should also be `--noinput-text=`

**Review by jeffmhubbard — CHANGES_REQUESTED** [review]
@nicolas-s Can you also fix `--noinput-text`?

**Comment by nicolas-s** [comment]
Ah yes I missed it.  I update the PR ASAP.

File: multilockscreen (hunk 1 above)
Comment (nicolas-s) [reply multilockscreen]: Fixed

**Review by jeffmhubbard — APPROVED** [review]

**Pull request merged**